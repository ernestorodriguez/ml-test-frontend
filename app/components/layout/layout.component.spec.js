const React = require('react');
const Layout = require('./index');
const Enzyme = require('enzyme');
const { shallow } = require('enzyme');

describe('Layout component', () => {
    test('should match snapshot', () => {
        const config = {
            title: 'Page Title'
        };

        const wrapper = shallow(<Layout {...config} />);

        expect(wrapper).toMatchSnapshot();
    });
});