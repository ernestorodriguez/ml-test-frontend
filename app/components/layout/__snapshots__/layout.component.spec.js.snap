// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should match snapshot 1`] = `
<Fragment>
  <head>
    <meta
      charSet="utf-8"
    />
    <title>
      Page Title
    </title>
    <meta
      content=""
      name="description"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      content="#fafafa"
      name="theme-color"
    />
  </head>
  <body>
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "
                <!--[if IE]>
                    <p class=\\"browserupgrade\\">
                        You are using an <strong>outdated</strong> browser.
                        Please <a href=\\"https://browsehappy.com/\\">upgrade your browser</a> to improve your experience and security.
                    </p>
                <![endif]-->",
        }
      }
    />
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "window.__PRELOADED_STATE__ =  {\\"pageData\\":undefined,\\"title\\":\\"Page Title\\"};",
        }
      }
    />
    <main
      id="root-app"
    />
  </body>
</Fragment>
`;
